controllers/disruption_controller.go

```diff
@@ -241,8 +242,14 @@ func (r *DisruptionReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 			return ctrl.Result{}, nil
 		}
 	} else {
-		if err := r.validateDisruptionSpec(instance); err != nil {
-			return ctrl.Result{Requeue: false}, err
```

Comment:
Are we not going to be validating the spec? I know we do it in a webhook, do you think it is redundant to do it during reconcile loop?

Reply:
If I understand correctly, the validation occurs in the validation webhook. So no need to verify it in the reconcile loop.

Reply:
I did look back on the git blame, and I added this check years before we had the webhooks. It's now worse than redundant, it's actively harmful. If we somehow have a disruption with an invalid spec get past the webhooks, we actually can't delete it or remove it's finalizer via the reconciler, we'd have to do that all manually with kubectl